docs/store/datastore.md

```diff
@@ -259,6 +259,16 @@ import os
 os.environ["HADOOP_USER_NAME"] = "..."
 ```
 
+To register an HDFS datastore profile and use it as described in [Data store profiles](#data-store-profiles), use:
+```python
+DatastoreProfileHdfs(
+    name="my-hdfs",
+    host=os.getenv("HDFS_HOST"),
+    port=int(os.getenv("HDFS_PORT")),
+    http_port=int(os.getenv("HDFS_HTTP_PORT")),
```

Comment:
Looks like this example is from the `TestFeatureStoreSparkEngine` system test suite. There it makes sense to use the environment variables, but here we probably want to have some straight examples:
```suggestion
    host="localhost",
    port=9000,
    http_port=9870,
```